rust_src/src/image.rs

```diff
@@ -0,0 +1,16 @@
+//! Functions for image support on window system.
```

Comment:
Is it worth making a image_debug.rs or similar so the code is not littered with cfg checks?

Reply:
I don't know, tbh I don't understand why those are all needed. Maybe we should wait until there are more uses before settling on a pattern?

Reply:
They are all needed because the compiler will notice and complain about unused imports, functions, etc. when the cfg is not true.